### ISSUE TITLE
make aws config local

### ISSUE THREAD

**Issue by hohwille** [devonfw/ide#964]
devonfw-ide started supporting `aws` CLI tool (see issue #711).
However, I noticed that the configuration remains global in the users home directory (in `~/.aws/` it creates files `config` and `credentials`).
Assuming you have two different projects for different customers with two different installations of devonfw-ide and their own `aws` CLI installation.
Now, these two should not interfere with each other so I should be able to do `aws configure` for both of them with totally different AWS accounts.
However, currently configuring one of them also configures the other and changes affect both/all installations.

Instead, we should tell `aws` to read the configuration from somewhere under `${DEVON_IDE_HOME}/conf/` (e.g. in `aws` or `.aws` subfolder).
Hint: We have archived the same already for tools like `mvn` (maven) or `az` (azure CLI).

**Comment by cinnamon-coder-hub** [comment]
While looking at this issue, I stumbled upon a setup problem. On my computer the msi file won't be installed. 
It turned out, that `powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${windows_path_to_package}\aws-*-windows.msi /quiet'"` won't work on my computer, because the asterix sign  `*`  in `aws-*-windows.msi` won't be replaced by the version number. So I tried to fix this with [the PR 989](https://github.com/devonfw/ide/pull/989). During another free moment I'll be looking at the actual problem mentioned by @hohwille.
